give.php

```diff
@@ -249,6 +252,7 @@ public function init() {
 	private function bindClasses() {
 		$this->container->singleton( 'templates', Templates::class );
 		$this->container->singleton( 'routeForm', FormRoute::class );
+		$this->container->singleton( LogUncaughtGiveExceptions::class );
```

Comment:
Is there a particular reason why this is named as a verb, instead of a noun, ie `UncaughtExceptionLogger`?

Reply:
Because when I wrote it I wasn't sure if I going making an Action class or not. It's not, so I'll rename. 😃

Reply:
Resolved in 0e71cc4